src/julia/Prechelt/src/Prechelt.jl

```diff
@@ -0,0 +1,76 @@
+#=
+# Port of Peter Norvig's Common Lisp program from http://norvig.com/java-lisp.html.
+#
+# - Julia version: 1.6.2
+# - Author: Renato Athaydes
+# - Date: 2021-07-24
+=#
+module Prechelt
+
+export run_encodings
+
+const emptyStrings = String[]
+
+const KEYS = begin
+    m = [
+         'j' 'r' 'd' 'f' 'a' 'c' 'b' 'l' 'g'
+         'n' 'w' 's' 't' 'm' 'i' 'k' 'o' 'h'
+         'q' 'x' 'y' '_' '_' 'v' 'u' 'p' 'z'
+        ]
+    map('a':'z') do c
+        n = 1
+        for col in eachcol(m) if c in col return n else n += 1 end end
+        0
+    end
+end
```

Comment:
Does this actually improve performance??
It'd better be a lot faster, because it's just so much less readable :D

Reply:
Yes, it's faster. You build an array once and index it when you need a digit, instead of searching for the digit every time.

An alternative would be to use the original `charToDigit` function and an array comprehension:
```julia
const KEYS = [charToDigit(c) for c in 'a':'z']
```
Though I didn't find it hard to read the matrix/map 🥲. Go through the columns*, if the character is in the column, return the index of the column.

I should've probably named it `i` instead of `n`.

*Julia is column-major

Reply:
Maybe this is more readable:

```julia
const KEYS = begin
    m = [
         'j' 'r' 'd' 'f' 'a' 'c' 'b' 'l' 'g'
         'n' 'w' 's' 't' 'm' 'i' 'k' 'o' 'h'
         'q' 'x' 'y' '_' '_' 'v' 'u' 'p' 'z'
        ]
    map('a':'z') do ch
        for i in 1:size(m, 2)
            ch in m[:, i] && return i
        end
        0
    end
end
```